supplier_app/tests/test_view.py

```diff
@@ -151,13 +157,16 @@ def test_get_success_url_should_redirect_to_home(self):
             )
 
     def _get_taxpayer_creation_request(self):
-        request = self.factory.get(
+        request = self.factory.post(
```

Comment:
should we consider changing the name of the function?

Reply:
I change it but I don't know what is wrong with the name